packages/devtools/lib/src/profiler/cpu_profile_tables.dart

```diff
@@ -82,7 +82,7 @@ class CpuBottomUp extends CpuProfilerView {
 
   @override
   void rebuildView() {
-    final CpuProfileData data = getProfileData();
+    final CpuProfileData data = profileData();
```

Comment:
there is a little bit of a confusing mixture of 
profileData() methods and getters. Maybe make them all getters?
In general, if getting the data is slow, a method should be used. Otherwise using a gett is cleaner.

Reply:
renamed to `profileDataProvider` to make it clear that this is a provider.